apps/src/blockly/addons/cdoXml.js

```diff
@@ -74,6 +75,50 @@ export default function initializeBlocklyXml(blocklyWrapper) {
   blocklyWrapper.Xml.createBlockOrderMap = createBlockOrderMap;
 }
 
+/**
+ * Adds a mutation element to a block if it should have an open miniflyout.
+ * CDO Blockly uses an unsupported method for serializing miniflyout state
+ * where arbitrary block attribute could be used to manage extra state.
+ * Mainline Blockly expects a mutator. The presence of the mutation element
+ * will trigger the block's domToMutation function to run, if it exists.
+ *
+ * @param {Element} blockElement - The XML element for a single block.
+ */
+export function addMutationToMiniToolboxBlocks(blockElement) {
+  const miniflyoutAttribute = blockElement.getAttribute('miniflyout');
+  if (!miniflyoutAttribute) {
```

Comment:
nit: Can we use `miniFlyout` since we seem standardized elsewhere on the code on `miniToolbox`? (I don't see any other mentions of `miniFlyout` in the code before this.)

Reply:
Are you suggesting renaming `miniflyoutAttibute` to `miniFlyout` or something else?

Reply:
No, sorry for the confusion! This was a super nit on the camel casing: `miniflyoutAttribute` -> `miniFlyoutAttribute`.

Reply:
I think we could swap casing here since we're moving between two different systems, but I think I prefer leaving it as is. `miniflyoutAttribute` refers directly to the attribute named `miniflyout` as opposed to just an attribute about _a mini-flyout_. If you have a strong opinion here, I'm happy to discuss further!